lambdas/tests/unit/handlers/conftest.py

```diff
@@ -62,3 +73,13 @@ def missing_id_event():
         "queryStringParameters": {"invalid": ""},
     }
     return api_gateway_proxy_event
+
+
+@pytest.fixture
+def mock_context():
+    class Context:
+        pass
+
+    context = Context()
+    context.aws_request_id = "fake_aws_request_id"
+    return context
```

Comment:
There is already a context fixture in `unit/conftest`, can you use it instead?

Reply:
done